### PR TITLE
invoice creation module

### PR DESCRIPTION
**Pull request by hadfl**

### CODE REVIEW

File: lib/OOCEapps/Model/Invoice.pm [hunk 1]
```diff
@@ -16,13 +15,27 @@ has schema => sub {
...
             validator   => $sv->regexp(qr/^.*$/, 'expected a string'),
         },
+        email_bcc => {
+            description => 'email bcc address',
+            example     => 'patrons@omniosce.org',
+            validator   => $sv->regexp(qr/^.*$/, 'expected a string'),
+        },
+        key_path => {
+            description => 'path to file containing the secret key',
+            example     => '/etc/opt/ooce/private/stripe.key',
```
Comment (oetiker) [inline lib/OOCEapps/Model/Invoice.pm]: here is it not stripe I think

File: templates/invoice/invoice.tex.ep [hunk 2]
```diff
@@ -36,7 +37,7 @@ patrons@omniosce.org
 \end{minipage}
 
 \vspace{4ex}
-<%== localtime->strftime('Olten, %e %B %Y') %>
+<%== Time::Piece->new($date)->strftime('Olten, %e %B %Y') %>
```
Comment (oetiker) [inline templates/invoice/invoice.tex.ep]: if you use Time::Piece I would also `use Time::Piece`

Comment (hadfl) [reply templates/invoice/invoice.tex.ep]: there is a `use Time::Piece` in line # 1

File: lib/OOCEapps/Model/Invoice.pm (hunk 1 above)
Comment (hadfl) [reply lib/OOCEapps/Model/Invoice.pm]: fixed naming in example

**Pull request merged**